opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -72,9 +95,25 @@ ENV CATALINA_HOME /usr/share/tomcat
 ENV PATH $CATALINA_HOME/bin:$PATH
 WORKDIR $CATALINA_HOME
 
+ARG USER_ID=${TOMCAT_USER_ID:-1000}
+ARG GROUP_ID=${TOMCAT_GROUP_ID:-1000}
+
+# Tomcat user and group created again on actual image definition to run as user.
+RUN    groupadd -r tomcat -g ${GROUP_ID}
+RUN    useradd -u ${USER_ID} -g tomcat -d ${CATALINA_HOME} -s /sbin/nologin \
+        -c "Tomcat user" tomcat
+
+# Change CATALINA_HOME ownership to tomcat user and tomcat group on actual definition
+RUN   chown -R tomcat:tomcat ${CATALINA_HOME}
+RUN   chmod -R 400 ${CATALINA_HOME/conf}
+
 HEALTHCHECK CMD curl --fail http://localhost:8081/healthcheck || exit 1
 
+ENV RUNAS_USER=tomcat
+
+CMD ["whoami"]
```

Comment:
Is this to be removed?

Reply:
No, that is what changes the user to tomcat, why would it be removed?

Reply:
That's a misunderstanding on your part.  That command doesn't change the user.

Reply:
Ah okay so in the read me it states
 'ENV RUNAS_USER=my-user
CMD ["whoami"] # Outputs my-user' 
However the CMD ["whoami"]  represents a placeholder for the CMD which in this case would be 'CMD ["catalina.sh", "run"]' ?

Reply:
It is just demonstrating the specified command is running as the user specified by `RUNAS_USER`.